main.R

```diff
@@ -136,6 +136,42 @@ if (nrow(techexposure_data) > 0) {
 techexposure_data
   }
 
+prep_exposure_stats <- function(audit_file, investor_name, portfolio_name, pacta_sectors) {
+  audit_table <- pacta.portfolio.report:::prep_audit_table(
+      audit_file,
+      investor_name = investor_name,
+      portfolio_name = portfolio_name,
+      currency_exchange_value = currency_exchange_value
+    )
+  
+  exposure_stats <- audit_file %>%
+      filter(.data$investor_name == .env$investor_name &
+        .data$portfolio_name == .env$portfolio_name) %>%
+      filter(.data$asset_type %in% c("Bonds", "Equity")) %>%
```

Comment:
I don't think this filtering is necessary, as you filter for `asset_type_analysis` later on (and all grouped analysis is grouped by `asset_type)

Reply:
I am not sure if I understand what you mean. You mean that we could keep all asset types in?

Reply:
Oops, sorry I meant to only highlight line 150

Reply:
Upon visually inspecting the code, this filter seems superfluous:
`filter(.data$asset_type %in% c("Bonds", "Equity")) %>%`

(I think)

Reply:
I understood which line you were referring to but I don't understand why you think it is superfluous.

Reply:
@jdhoffa since we want only to produce data for the asset classes for which we perform PACTA analysis I think it is relevant that we filter on those. Especially in the light of the code I added after your initial review.

Reply:
Previously you had two separate filters for the same entities I believe! Which is why I mentioned it wasn't necessary. 

Resolving since it is no longer relevant :)